src/components/LazyLog/README.md

```diff
@@ -35,3 +35,29 @@ consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?
   <LazyLog extraLines={1} enableSearch text={text} caseInsensitive />
 </div>
 ```
+
+Log viewing using a websocket
+
+```jsx harmony
+const url = 'wss://echo.websocket.org';
+let socket = null;
+
+<div style={{ height: 200, width: 902 }}>
+  <button style={{margin: 5}} onClick={() => socket && socket.send(JSON.stringify({ message: '[taskcluster 2018-11-14 21:08:32.452Z] Worker Group: us-east-1' }))}>ping</button>
+  <LazyLog
+    enableSearch
+    url={url}
+    websocket
+    websocketOptions={{
+      onOpen: (e, sock) => {
+        console.log("opened");
```

Comment:
We can remove the `console.log` statement.

Reply:
👍